.github/workflows/cron-denylist.yml

```diff
@@ -0,0 +1,25 @@
+name: Cron sync deny list
+
+on:
+  schedule:
+    - cron: '13 8 * * *'
+  workflow_dispatch:
+
+jobs:
+  update:
+    name: Sync deny list with KV store
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        env: ['staging', 'production']
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - uses: bahmutov/npm-install@v1
+      - name: Run job
+        env:
+          CF_API_TOKEN: ${{ secrets.CF_API_TOKEN }}
```

Comment:
```suggestion
          CF_API_TOKEN: ${{ secrets.CF_GATEWAY_TOKEN }}
```

Reply:
We use a new token since we moved to nftstorage.link zone

Reply:
Ah yes, ty